README.md

```diff
@@ -16,7 +16,7 @@ For storage, you should always try to use the `pseudo_iban`, falling back to `ib
 
 For example:
 - Sweden does support IBANs (**1.**) but the format is ambiguous due to variable length account numbers so they cannot be deconstructed (**2.**). For persistence, we therefore recommend using pseudo-IBANs (**3.**) because the national banking details can be recovered from them.
-- Australia does not support IBANs (**1.** & **2.**), therefore pseudo-IBANs (**3.**) can be created from national banking details for storage. To get back the national banking details, you can pass the pseudo-IBAN to Ibandit and it will parse out the national banking details again for use.
+- Australia (and New Zealand) do not support IBANs (**1.** & **2.**), therefore pseudo-IBANs (**3.**) can be created from national banking details for storage. To get back the national banking details, you can pass the pseudo-IBAN to Ibandit and it will parse out the national banking details again for use.
```

Comment:
Nitpick: not necessary. This is designed to explain one example of each, not be an exhaustive list. The table below is the exhaustive version.